docker-compose.yml.template

```diff
@@ -50,6 +50,8 @@ services:
       # Default to caller's environment
       AWS_ACCESS_KEY_ID:
       AWS_SECRET_ACCESS_KEY:
+      # Needed to resolve env info
+      GLOBAL_ENV_BUCKET:
```

Comment:
Why no value? Shouldn't it be the cgap devtest global bucket env, `cgap-devtest-main-foursight-envs`?